extensions/amp-app-banner/0.1/test/test-amp-app-banner.js

```diff
@@ -270,8 +276,17 @@ describe('amp-app-banner', () => {
       });
     });
 
-    it('should remove banner if safari', () => {
+    it('should remove banner if safari and not embedded', () => {
```

Comment:
hmmm according to the description here and below we will remove banner as long as it is safari? Really